### ISSUE TITLE
Missing ossec-* packages in base-64-lts 20.4.0 ?

### ISSUE THREAD

**Issue by dca58** [TritonDataCenter/pkgsrc#294]
Those packages are missing from 2020Q4, will they be added later or is ossec being abandoned ?
Particularly interested in the ossecc-client package as it is running in all of our zones.

**Comment by jperkin** [comment]
I'll fix those shortly, thanks for the headsup!

They also haven't been updated for a while, I don't use these packages personally, so assume they are ok to update to the latest stable 3.6.0 version?

**Comment by dca58** [comment]
Yes, that would be perfect, we already have a few 3.6 instances on Ubuntu connected to a 2.9 server.

**Comment by jperkin** [comment]
It'll take a while to update to the latest, so for now I've just fixed the existing packages.  They are now available in 2020Q4.

I'll leave this ticket open for the update.

**Comment by dca58** [comment]
There are indeed there, thanks a lot, that was quick !
